src/Features/CSharp/Portable/Completion/CompletionProviders/XmlDocCommentCompletionProvider.cs

```diff
@@ -286,6 +286,7 @@ private IEnumerable<CompletionItem> GetTagsForMethod(IMethodSymbol symbol, TextS
             }
 
             items.AddRange(parameters.Select(p => CreateCompletionItem(itemSpan, FormatParameter(ParamTagName, p))));
+            items.AddRange(parameters.Select(p => CreateCompletionItem(itemSpan, FormatParameter(ParamRefTagName, p))));
```

Comment:
I realize that using the same `parameters` for both "param" and "paramref" doesn't make sense here, since some parameters are filtered out. I will push a fix soon.